projects/bp-gallery/cypress/e2e/bulk-edit/collections.cy.ts

```diff
@@ -0,0 +1,119 @@
+import { login, logout } from '../../utils/login-utils';
+import { selectPictures, visitArchive1Pictures } from './helper';
+
+describe('bulk edit collections', () => {
+  before(() => {
+    cy.visit('/');
+    login();
+  });
+  after(() => {
+    cy.visit('/');
+    logout();
+  });
+
+  it('shows the intersection of the given collections', () => {
+    cy.visit('/picture/3');
+    cy.contains('Collection 1');
+    cy.contains('Collection 2');
+    cy.visit('/picture/2');
+    cy.contains('Collection 1');
+    cy.contains('Collection 2').should('not.exist');
+    visitArchive1Pictures();
+    selectPictures('3', '2');
+    cy.contains('Mehrere Bilder editieren').click();
+    cy.contains('Collection 1');
+    cy.contains('Collection 2').should('not.exist');
+  });
+
+  it("shows no collections when one picture doesn't have any collections", () => {
+    // create a picture without collections
+    cy.visit('/picture/1');
+    cy.contains('.MuiChip-root', 'Top-Level collection').find('[data-testid="CancelIcon"]').click();
+    cy.contains('.save-state', 'Gespeichert');
+
+    cy.visit('/picture/3');
+    cy.contains('Collection 1');
+    cy.contains('Collection 2');
+    cy.visit('/picture/1');
+    cy.contains('.picture-info-field', 'Collections').find('input');
+    cy.contains('.picture-info-field', 'Collections').find('.MuiChip-root').should('not.exist');
+    visitArchive1Pictures();
+    selectPictures('3', '1');
+    cy.contains('Mehrere Bilder editieren').click();
+    cy.contains('.picture-info-field', 'Collections').find('input');
+    cy.contains('.picture-info-field', 'Collections').find('.MuiChip-root').should('not.exist');
+
+    // readd the deleted collection
```

Comment:
kommentar irreführend, ihr setzt doch hier die Collection wieder

Reply:
Wir fügen hier die Collection zu der Liste der Collections, in denen das Bild ist, wieder hinzu. Hast du eine bessere Formulierung? Mir würde noch sowas einfallen:
```ts
// readd the picture to the collection, from which we removed it above
```